src/components/UI/__tests__/cluster_dashboard_item.js

```diff
@@ -0,0 +1,45 @@
+import 'jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import React from 'react';
+import ClusterDashboardItem from 'UI/cluster_dashboard_item';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  render(<ClusterDashboardItem />, div);
+});
+
+it('has links to the cluster detail page', () => {
```

Comment:
@ograu can we pair and implement these tests together? I'm not sure where to start.

Reply:
Yes, just ping me on slack please